lua/nvim-treesitter/query_predicates.lua

```diff
@@ -1,8 +1,19 @@
 local utils = require'nvim-treesitter.utils'
 local ts_utils = require'nvim-treesitter.ts_utils'
 
+local _ts_locals
 local M = {}
 
+-- Lazily have to load this to avoid circular dependencies.
+-- Cache it once it's loaded to a
+local function get_locals_api()
```

Comment:
Not sure how else to handle this one without moving a bunch of stuff around. Let me know if there might be a better way.

Reply:
I guess you can just `require` here performance implications will only be minimal. Require only runs the required file once: https://www.lua.org/pil/8.1.html

But for me this solution is also fine.